.changeset/slimy-files-feel.md

```diff
@@ -0,0 +1,5 @@
+---
+"@biomejs/biome": patch
```

Comment:
This can't be a patch. It's supposed to be a minor because it's a new feature :)